Example/Auth/Sample/MainViewController+OAuth.m

```diff
@@ -75,6 +79,68 @@ - (void)signInGoogleHeadfulLite {
   }];
 }
 
+- (void)linkWithGoogleHeadfulLite {
+  FIROAuthProvider *provider = self.googleOAuthProvider;
+  provider.customParameters = @{
+                                @"prompt" : @"consent",
+                                };
+  provider.scopes = @[ @"profile", @"email", @"https://www.googleapis.com/auth/plus.me" ];
+  [self showSpinner:^{
+    [[AppManager auth].currentUser linkWithProvider:provider
+                                         UIDelegate:nil
+                                         completion:^(FIRAuthDataResult *_Nullable authResult,
+                                                      NSError *_Nullable error) {
+     [self hideSpinner:^{
+       if (error) {
+         [self logFailure:@"Reauthenticate with provider (Google) failed" error:error];
+       } else if (authResult.additionalUserInfo) {
+         [self logSuccess:[self stringWithAdditionalUserInfo:authResult.additionalUserInfo]];
+         if (self.isNewUserToggleOn) {
+           NSString *newUserString = authResult.additionalUserInfo.newUser ?
+           @"New user" : @"Existing user";
+           [self showMessagePromptWithTitle:@"New or Existing"
+                                    message:newUserString
+                           showCancelButton:NO
+                                 completion:nil];
+         }
+       }
+       [self showTypicalUIForUserUpdateResultsWithTitle:@"Link Error" error:error];
+     }];
+   }];
+  }];
+}
+
+- (void)reauthenticateWithGoogleHeadfulLite {
+  FIROAuthProvider *provider = self.googleOAuthProvider;
+  provider.customParameters = @{
+                                @"prompt" : @"consent",
+                                };
+  provider.scopes = @[ @"profile", @"email", @"https://www.googleapis.com/auth/plus.me" ];
+  [self showSpinner:^{
+    [[AppManager auth].currentUser reauthenticateWithProvider:provider
+                                                   UIDelegate:nil
+                                                   completion:^(FIRAuthDataResult *_Nullable authResult,
+                                                                NSError *_Nullable error) {
+     [self hideSpinner:^{
+       if (error) {
+         [self logFailure:@"Link with provider (Google) failed" error:error];
+       } else if (authResult.additionalUserInfo) {
+         [self logSuccess:[self stringWithAdditionalUserInfo:authResult.additionalUserInfo]];
+         if (self.isNewUserToggleOn) {
+           NSString *newUserString = authResult.additionalUserInfo.newUser ?
+           @"New user" : @"Existing user";
+           [self showMessagePromptWithTitle:@"New or Existing"
```

Comment:
Does this need to be localized?

Reply:
It's the sample app so we don't localize it.